tds_test.go

```diff
@@ -520,3 +521,37 @@ func TestBadConnect(t *testing.T) {
 		t.Error("Ping should fail for connection: ", badDSN)
 	}
 }
+
+func TestSSPIAuth(t *testing.T) {
+	if runtime.GOOS != "windows" {
+		t.Skip("Only on windows")
+	}
+	checkConnStr(t)
+	connStr := makeConnStr(t)
+	params := connStr.Query()
+	params.Set("Integrated Security", "sspi")
+	connStr.RawQuery = params.Encode()
+
+	conn, err := sql.Open("mssql", connStr.String())
```

Comment:
Do not call this `conn`. This is not a database connection, this represents a database pool that hasn't made a single connection.

Prefer `db` or `pool`.

The reason why this name is important is people read examples and testing code and sometimes mis-understand what is going on, then go on to write incorrect or poor quality code (like creating a connection pool for every query).